### PR TITLE
updated sidebar filter to include crop groups

### PR DESCRIPTION
**Pull request by Adamws33**
Updated the sidebar to include a filter for cover crop group.

### CODE REVIEW

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://52.224.128.96:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://52.224.128.96:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

File: src/pages/CropSidebar/CropSidebar.jsx [hunk 1]
```diff
@@ -353,6 +367,38 @@ const CropSidebar = ({
...
+          return (
+            <Grid key={val.label} item>
+              <Chip
+                key={val.label}
+                onClick={() => handleCropGroupFilter(val.label)}
+                component="li"
+                size="medium"
+                label={val.label}
+                color={selected ? 'primary' : 'secondary'}
+              />
```
Comment (YazidZidane) [inline src/pages/CropSidebar/CropSidebar.jsx:397]: Can you add some gap between the chips, just like other chip filters?
image

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://52.224.128.96:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://52.224.128.96:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Review by YazidZidane — APPROVED** [review]

**Pull request merged**